lib/tasks/codereview.rake

```diff
@@ -0,0 +1,11 @@
+require 'pronto'
+
+desc "codereview"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.